plasma_framework/python_tests/tests/tests_utils/plasma_framework.py

```diff
@@ -343,6 +343,9 @@ def childBlockInterval(self):
     def standardExitBond(self):
         return self.payment_exit_game.startStandardExitBondSize()
 
+    def processStandardExitBounty(self):
+        return 107000 * 100
```

Comment:
where does this 100 come from?? Also, I think we should make this the same as `standardExitBond` which gets data from contract directly